internal/casclient/casclient.go

```diff
@@ -24,8 +24,8 @@ import (
 )
 
 type UpDownStatus struct {
-	Filepath, Filename, Digest string
```

Comment:
`Filepath` was not actually used and added an unnecessary dependency to a local file